src/main.ts

```diff
@@ -58,10 +58,21 @@ type tailscaleStatus = {
 
 // Cross-platform Tailscale local API status check
 async function getTailscaleStatus(): Promise<tailscaleStatus> {
-  const { stdout } = await exec.getExecOutput(cmdTailscale, [
-    "status",
-    "--json",
-  ]);
+  const { exitCode, stdout, stderr } = await exec.getExecOutput(
+    cmdTailscale,
+    ["status", "--json"],
+    {
+      silent: true,
+      ignoreReturnCode: true,
+    }
+  );
+  if (exitCode !== 0) {
+    process.stderr.write(stderr);
+    throw new Error(`tailscale status failed with exit code ${exitCode}`);
+  }
+  if (core.isDebug()) {
+    process.stdout.write(stdout);
```

Comment:
We do a `core.debug` call  with the result of `getTailscaleStatus` later down the line, does this operate differently or should we remove one of these calls?

Reply:
The later one shows round-tripped JSON that's not even pretty-printed, which isn't very useful. I'll just remove that one.

Reply:
Removed.